setrpaths.sh

```diff
@@ -134,7 +134,19 @@ fi
 
 SAVEIFS=$IFS
 IFS=$(echo -en "\n\b")
-for filename in $(find $ARG_PATH -type f); do
+for filename in $(find $ARG_PATH -type f -print0 | python -c '
```

Comment:
could filter `.whl` and `.jar` at the `find` level instead of in python, with `-not -name '*.whl' -a -not -name '*.jar'`

Reply:
we could also speed it up by excluding other file types, like `.html`, `.txt`, etc.

Reply:
hmm we want to specifically *include* whl and jar files and no look at them, not exclude them.
I can however restructure it with the Python code returning an exit code and `find` testing for that.

excluding by extension can be a bit tricky, e.g. for MATLAB most end in `.m` but there may be some other program that has executables ending in `.m`, so I'd rather not go down that rabbit hole.

Reply:
ah, right, it would require two finds (one for jar/whl and one for the rest). Disregard my comment.